src/modules/75-cf/pages/feature-flags/FlagStatus.tsx

```diff
@@ -22,19 +22,27 @@ export enum FeatureFlagStatus {
   RECENTLY_ACCESSED = 'recently-accessed',
   ARCHIVED = 'archived'
 }
+
+export enum StaleFlagStatusReason {
+  WAITING_FOR_CLEANUP = 'marked_by_user',
+  POTENTIALLY_STALE = 'matched_criteria'
+}
+
 export interface FlagStatusProps {
   status?: FeatureFlagStatus
   lastAccess?: number
-  stale?: boolean
+  staleReason?: string
 }
 
-export const FlagStatus: React.FC<FlagStatusProps> = ({ status, lastAccess, stale }) => {
+export const FlagStatus: React.FC<FlagStatusProps> = ({ status, lastAccess, staleReason }) => {
   const { getString } = useStrings()
   const isNeverRequested = status === FeatureFlagStatus.NEVER_REQUESTED
   const isPotentiallyStale = status === FeatureFlagStatus.POTENTIALLY_STALE
 
   const flagCleanupEnabled = useFeatureFlag(FeatureFlag.FFM_8344_FLAG_CLEANUP)
 
+  const isWaitingForCleanup = staleReason == StaleFlagStatusReason.WAITING_FOR_CLEANUP
```

Comment:
Can you use strict comparison here?
```suggestion
  const isWaitingForCleanup = staleReason === StaleFlagStatusReason.WAITING_FOR_CLEANUP
```

Reply:
Thanks! Have updated